docs/organizations/members.md

```diff
@@ -1,52 +1,65 @@
+![Organization New Invite](/images/organizations/organization-new-invite.png "Organization New Invite")
 
 Pending invitations are limited by the maximum number of members your organization can have. For example, if your organization can have a maximum of five members, and the organization already has three members, you may only have two pending invitations open at any time.
 
-Once all of your invitations are ready, click the `Send Invitations` button at the bottom of the list. Each of your invitees will receive an email with instructions on how to join your organization. If they do not already have a Losant account, they can create one as part of the invitation acceptance process.
+Each of your invitees will receive an email with instructions on how to join your organization. If they do not already have a Losant account, they can create one as part of the invitation acceptance process.
 
-To revoke a pending invitation - whether it has already been sent or not – click the `Remove` (-) icon to the right of the invitee's info. If you need to change an invitee's role, you will either have to wait for the user to accept the invitation and then change their access, or you will have to revoke the current invitation and send the user a new one.
+Once a user is invited, administrators may view the details of the invitation from the `Invite Details` screen. This can be accessed by clicking on the email of the invitee from either the `Organization Overview` or `Members` screen.
 
-![Organization Expired Invites](/images/organizations/organization-expired-invites.png "Organization Expired Invites")
+![Organization Invite Details](/images/organizations/organization-invite-details.png "Organization Invite Details")
+
+To revoke a pending invitation, click `Revoke Invitation` to the right of the invited user's email and confirm this action. Administrators may also revoke any invitation from the `Members` screen by clicking the `Remove` icon in the furthest right table column. If you need to change any of the invitee's roles, you will either have to wait for the user to accept the invitation and then change their access, or you will have to revoke the current invitation and send the user a new one.
 
 ## Expired Invitations
 
-Any invitation that was not accepted after seven days appears in the `Expired Invitations` section of the `Members` page. Invitations in this state do not count against your new invitation limit. Invitees whose invitation has expired can no longer join your organization.
+Any invitation that was not accepted after seven days has expired. Invitations in this state do not count against your new invitation limit. Invitees whose invitation has expired can no longer join your organization.
```

Comment:
I'd clarify this a bit; they can no longer join the organization *using this invitation*; they can always be sent a new invitation (or a renewal).